cypress/utils/plugins/observability-dashboards/constants.js

```diff
@@ -44,8 +44,8 @@ export const supressResizeObserverIssue = () => {
 };
 
 export const setTimeFilter = (setEndTime = false, refresh = true) => {
-  const startTime = 'Mar 25, 2021 @ 10:00:00.000';
-  const endTime = 'Mar 25, 2021 @ 11:00:00.000';
+  const startTime = 'Mar 25, 2021 @ 17:00:00.000';
+  const endTime = 'Mar 25, 2021 @ 18:00:00.000';
```

Comment:
Does having a bigger window like 2 days here, solve covering all the time-zones? 
```
Mar 24, 2021 @ 17:00:00.000 -> Mar 26, 2021 @ 17:00:00.000
```